homeassistant/components/conversation/agent.py

```diff
@@ -25,6 +25,10 @@ async def async_set_onboarding(self, shown):
 
     @abstractmethod
     async def async_process(
-        self, text: str, context: Context, conversation_id: str | None = None
+        self,
+        text: str,
+        context: Context,
+        conversation_id: str | None = None,
+        language: str | None = None,
```

Comment:
Should we mark this as a breaking change

Reply:
Looks like we should, since almond broke on it.